pep8.py

```diff
@@ -1933,7 +1948,7 @@ def get_parser(prog='pep8', version=__version__):
     parser.config_options = [
         'exclude', 'filename', 'select', 'ignore', 'max-line-length',
         'hang-closing', 'count', 'format', 'quiet', 'show-pep8',
-        'show-source', 'statistics', 'verbose']
+        'show-source', 'statistics', 'verbose', 'stdin-name']
```

Comment:
@sigmavirus24, True.

I was under the mistaken assumption that it was necessary for cmdline switches as well. Will fix.